src/editor-model/array.ts

```diff
@@ -137,6 +137,11 @@ function addCell(
         break;
 
       case 'after column':
+        if (
+          arrayAtom.colFormat.filter((col) => Boolean(col['align'])).length <=
+          arrayAtom.colCount
+        )
+          return;
```

Comment:
Could do a `model.announce('plonk')` before `return`. It's used to indicate to acknowledge the command, and to indicate the command is not currently applicable. It plays a "plonk" sound when the sound file is available.